tests/publiccloud/download_repos.pm

```diff
@@ -63,22 +63,22 @@ sub run {
                 record_soft_failure("Download failed (rc=$ret):\n$maintrepo");
                 script_run("echo 'Download failed for $maintrepo ...' >> ~/repos/qem_download_status.txt");
             } else {
-                assert_script_run("echo -en '# $maintrepo:\\n\\n' >> /tmp/repos.list.txt");
-                script_run("echo 'Downloaded $maintrepo: `du -hs $maintrepo`' >> ~/repos/qem_download_status.txt");
+                assert_script_run("echo -en '\\n" . ('#' x 80) . "\\n# $maintrepo:\\n' >> /tmp/repos.list.txt");
+                assert_script_run("echo 'Downloaded $maintrepo:' \$(du -hs $parent | cut -f1) >> ~/repos/qem_download_status.txt");
                 if (script_run("ls $parent*.repo") == 0) {
                     assert_script_run("sed -i \"1 s/\\]/_\$(head /dev/urandom | tr -dc A-Za-z0-9 | head -c 4)]/\" $parent*.repo");
```

Comment:
I find this line horrible. Perhaps we should refactor it in a future PR.

Reply:
What you could do is immediately create a ticket for the idea or maybe even better a (draft) PR to catch the idea as this is certainly out of scope for this PR

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/12268